scripts/build/golines.sh

```diff
@@ -44,7 +44,7 @@ fi
 # Find all .go files in the project directory and its subdirectories, ignoring .pb.go and .pb_encoding.go files
 find "${ROOT_DIR}" -type f -name "*.go" ! -name "*.pb.go" ! -name "*.pb_encoding.go" | while read -r file; do
     echo "Processing $file..."
-    golines --reformat-tags --shorten-comments --write-output --max-len=80 "$file"
+    golines --reformat-tags --shorten-comments --write-output --max-len=140 "$file"
```

Comment:
@fridrik01 I found the culprit. This was shortening lines whenever I ran `make format` locally which runs this golines script